libvast/src/schema.cpp

```diff
@@ -219,23 +220,34 @@ caf::expected<schema> get_schema(const caf::settings& options) {
 
 detail::stable_set<std::filesystem::path>
 get_schema_dirs(const caf::actor_system_config& cfg,
-                std::vector<const void*> objpath_addresses) {
+                const std::vector<const void*>& objpath_addresses) {
   const auto disable_default_config_dirs
     = caf::get_or(cfg, "vast.disable-default-config-dirs", false);
   detail::stable_set<std::filesystem::path> result;
   if (auto vast_schema_directories = detail::locked_getenv("VAST_SCHEMA_DIRS"))
     for (auto&& path : detail::split(*vast_schema_directories, ":"))
       result.insert({path});
+  auto insert_dirs_for_datadir = [&](const std::filesystem::path& datadir) {
+    result.insert(datadir / "schema");
+    for (const auto& plugin : plugins::get()) {
+      std::error_code err{};
+      auto dir = datadir / "plugin" / plugin->name() / "schema";
+      if (auto err = std::error_code{}; std::filesystem::exists(dir, err))
```

Comment:
You define `err` twice. (personally the variant where it's defined outside the if-statement looks a bit easier to read)